doc/design/initialization.md

```diff
@@ -8,41 +8,6 @@ PaddlePaddle needs a way to init parameters.
 
 Initialization operators must run once and only one; otherwise, every iteration would clear the parameters.
 
-## Proposals
-
-### Proposal 1: Two seperate `ProgramDesc`.
+## Solution: Two seperate `ProgramDesc`.
```

Comment:
Who will output the two separate `ProgramDesc`s?

I my mind:

Python ---(entire `ProgramDesc`) ---> Pruner ---(ProgramDesc1, ProgramDesc)---> Executer.

We need to make sure Pruner have the global view. For example, in cluster training, the Pruner need to have the global view in order to do things like deciding parameter placement.

Reply:
Python generates two separate `ProgramDesc`, one for initialization, and one for training. Both `ProgramDesc`s contain all the parameter description. Do you think it is enough to decide parameter placement? If not, we can always send both `ProgramDesc`s to Pruner.

Reply:
If Python can do the separation, pruner should be able to do it as well. Is it possible to let pruner do the job?

Parameter placement is just an example supporting the argument that the pruner needs to have the global view.